lib/rubocop/cop/rails/unique_validation_without_index.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # When you define uniqueness validation in Active Record model,
+      # you also should add a unique index for the column. It has two reasons.
```

Comment:
'It has two reasons' reads a little awkward – maybe change to 'There are two reasons' or 'There are two reasons for this'?